test/optimise.jl

```diff
@@ -73,6 +85,12 @@ using Test
             @test global_max ⊆ 199.9..200
             m = (9.99..10)
             @test all(X ⊆ m × m || X ⊆ -m × m || X ⊆ m × -m || X ⊆ -m × -m for X in maximisers)
+
+            # same but with BigFloats
+            global_min, minimisers = minimise( X -> ( (x,y) = X; x^2 + y^2 ), (-big(10.0)..big(10.0)) × (-big(10.0)..big(10.0)), structure = Structure )
+            @test global_min ⊆ 0..1e-7
+            @test all(X ⊆ big(-1e-3..1e3) × big(-1e-3..1e-3) for X in minimisers)
```

Comment:
We need to make to use `big(-1e-3..1e3)` here due to the type constraint here:

https://github.com/JuliaIntervals/IntervalArithmetic.jl/blob/1516c5e1ca8e088aeafa9574072512aa2df241fb/src/multidim/intervalbox.jl#L69

Note that for the 1D tests, eg `@test global_min ⊆ -10 .. -9.999` works even when `global_min` is a Bigfloat interval. I think that type constraint could be lifted.